monitor-agent/docker-compose.prod.yml

```
@@ -0,0 +1,67 @@
+    environment:
+      - DOMAIN_NAME=${DOMAIN_NAME}
+      - ALLOWED_IPS=${ALLOWED_IPS}
+    depends_on:
+      - monitor-agent
+    restart: always
+    networks:
+      - monitor-network
+
+volumes:
+  caddy_data:
+  caddy_config:
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)